tutoraspects/templates/aspects/apps/aspects/scripts/dbt.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+
+set -eo pipefail
```

Comment:
This allows command errors to bubble up and fail the Tutor command when there's an error